src/components/ComponentsPanel.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <q-drawer
```

Comment:
I prefere to have a separate component like DefaultDrawer, that's defined all default option.

And we can send to it a slot for the content and a property to say if we want it on the left, right or bottom.

Like that we can reuse it for another next component.